ci/Jenkinsfiles/cleanup-exports.groovy

```diff
@@ -96,7 +96,7 @@ pipeline {
             ----------------------------------------"""
             withCredentials([usernameColonPassword(credentialsId: TARGET_CREDS_ID, variable: 'EXPLORER_PASS')]) {
               def curlCommand = "curl --user ${EXPLORER_PASS} ${CURL_OPTIONS}"
-              def query = "SELECT * FROM NXDistribution WHERE nxdistribution:aliases='next' ORDER BY nxdistribution:released DESC"
+              def query = "SELECT * FROM NXDistribution WHERE nxdistribution:aliases='next' ORDER BY dc:created ASC"
```

Comment:
`ASC` can be removed, it's the default

Reply:
Yes but since i was previously confused about the right ordering, i wanted to keep this explicit